railties/test/command/base_test.rb

```diff
@@ -12,4 +12,22 @@ class Rails::Command::BaseTest < ActiveSupport::TestCase
     assert_equal %w(secrets:setup secrets:edit secrets:show), Rails::Command::SecretsCommand.printing_commands
     assert_equal %w(db:system:change), Rails::Command::Db::System::ChangeCommand.printing_commands
   end
+
+  test "ARGV is isolated" do
```

Comment:
This may not be the appropriate place for this test.  Should I create `railties/test/command_test.rb` and move this test there?